app/lambda/create_session/lambda_function.py

```diff
@@ -0,0 +1,65 @@
+import json
+import sys
+import time
+import uuid
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+
+
+class CreateSessionLambda(LambdaBase):
+    def __init__(self): 
+        pass
+
+    def CurTimeToEpoch(self): 
+        return int(time.time())
+
+    def EpochToCurTime(self, epoch): #int
+        return time.strftime('%Y-%m-%d %H:%M:%S', time.localtime(epoch))
+
+    def putItem(self, PatientId, HealthCareProfessionalId, SessionName, TimeStampStart):
+        client = boto3.resource('dynamodb')
+        table = client.Table("Sessions")
+        tempId = uuid.uuid4().hex 
+        epochTime = self.CurTimeToEpoch()
+        SessionId = "s-"+tempId+str(epochTime)
```

Comment:
Please review this again.
I believe the session id is supposed to be s-{epochtime}-{uuid}
Only then can we sort sessions per patient

Reply:
Done.

Reply:
As dicussed offline,we may need to revisit the algorithm/steps for CreateSession later